### PR TITLE
Fix Negative Number vs Negative Decimal and make 1 invalid case more clear and fix style guide

### LINKED ISSUE #6: Enforced Style Guide
https://github.com/RandomSoup/sack/blob/9ee412a53d09e1879d5f0a9ce1319b9db5943ab9/readme.md?plain=1#L26

For consistency between sack compilers, a selection of one or the other should probably be made (like a warning).

### LINKED ISSUE #7: Valid Number vs Valid Decimal
https://github.com/RandomSoup/sack/blob/9ee412a53d09e1879d5f0a9ce1319b9db5943ab9/readme.md?plain=1#L68

Shouldn't -0.5 be a valid Decimal instead of valid Number, as Number is an integer

### PR DESCRIPTION
**Pull request by StealthHydra179**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -99,10 +99,14 @@ let d = 0.0;
 The following are vaild negative Numbers:
```
Comment (nobody5050) [inline readme.md]: There’s a typo on this line that predates this pull, however it could probably be fixed in this pull.

**Review by nobody5050 — COMMENTED** [review]
Looks good

File: readme.md (hunk 1 above)
Comment (StealthHydra179) [reply readme.md]: What is the typo?

Comment (nobody5050) [reply readme.md]: vaild vs valid

Comment (StealthHydra179) [reply readme.md]: Fixed

File: readme.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ Sack is a dynamically typed scripting language for beginner programmers that foc
 	- Decimal (32 bit float)
 	- Bool (`true` or `false`)
 	- None (`none`)
-	- Identifier
+  - Identifier
```
Comment (nobody5050) [inline readme.md]: Why was the indent removed here?

File: readme.md [hunk 3]
```diff
@@ -19,11 +19,11 @@ Sack is a dynamically typed scripting language for beginner programmers that foc
...
 	- List (Itterable Scope)
 	- file (File object)
 	- byte (8 bit integer)
-- Sack enforces a style guide to improve readability, it is left up to the implementation whether this is a default warn or error. For simplicity, the style guide will not be described in this spec, however the example code will follow it.
+- Sack enforces a style guide for improve readability. Compilers should by default warn for violations of the style guide.
```
Comment (nobody5050) [inline readme.md]: “For improve readability?”

**Review by nobody5050 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (StealthHydra179) [reply readme.md]: Fixed

File: readme.md (hunk 3 above)
Comment (StealthHydra179) [reply readme.md]: Fixed

**Review by nobody5050 — APPROVED** [review]

**Pull request merged**